.buildkite/core.rayci.yml

```diff
@@ -286,6 +286,19 @@ steps:
     commands:
       - bazel run //ci/ray_ci:test_in_docker -- //... core
         --run-flaky-tests  --build-type clang
+        --except-tags multi_gpu
+
+  - label: ":ray: core: flaky gpu tests"
+    tags:
+      - gpu
+      - python
+      - skip-on-premerge
+    instance_type: gpu-largek
+    soft_fail: true
```

Comment:
they are already skipping on premerge, do we still need `soft_fail`? can we just remove all the soft fails on these flaky tests?

if test fails here, they should be blocking release, right?

Reply:
go/is-ray-green will block the releases ;); we still need them to soft fail because not all failing tests are release blocking; non-release blocking failing tests still run in these jobs

Reply:
> go/is-ray-green is a better source of truth

- https://buildkite.com/ray-project/postmerge/builds/4953#01901a79-80f5-413d-85db-510a68adfcf3 (test_accelerated_dag is consistently failing)
- release automation that triggered it is also showing green? : https://buildkite.com/ray-project/release-automation/builds/603
- including check release blocker? : https://buildkite.com/ray-project/release-automation/builds/603

how should I read the color in last night's case?

Reply:
> not all failing tests are release blocking

can the blocking ones and non-blocking ones be separated out? is the "release-blocking" property also saved in the test db?

Reply:
last night case was a human (basically me) tampering with the automation ;) - see https://github.com/orgs/anyscale/projects/76/views/1?pane=issue&itemId=67247006, it was me closing the issue by trusting the revert fixed the issue (the `blame reverted` comments)

the automation is doing the right thing, but a human was taking action without waiting for the automation to actually confirm the fix

Reply:
@aslonnie it can yes and you're more than welcomed to improve these logic ;), my personal opinion is though that is a wrong problem to solve, a status of a flaky test run cannot be reliably used as a release-blocking signal (because the tests are flaky)

Reply:
> a status of a flaky test run cannot be reliably used as a release-blocking signal (because the tests are flaky)

- yes, though for this particular case, it is not flaky. it is consistently failing? but it takes time to tell
- what are we using as the "release-blocking" signal today? or say if some one needs to implement it, what should s/he use?
- when it is truly hard to tell if a failing/flaky test is release blocking or not, should we be preferring on the wrongly-blocking good code side, rather on the wrongly-releasing bad code side?